pkg/services/provisioning/dashboards/types.go

```diff
@@ -25,6 +26,7 @@ type DashboardsAsConfigV0 struct {
 	Type                  string                 `json:"type" yaml:"type"`
 	OrgId                 int64                  `json:"org_id" yaml:"org_id"`
 	Folder                string                 `json:"folder" yaml:"folder"`
+	FolderUid             string                 `json:"folderUid" yaml:"folderUid"`
```

Comment:
Not sure about having this also in version 0. Seems weird adding new features to old version, on the other hand it is not breaking so not a big issue. cc @bergquist ?

Reply:
I think its fine as long as we dont introduce any breaking changes. In heinsight, we should never have two versions. Sometimes I spend to much time avoiding breaking changes ;)